src/utils/utils.ts

```diff
@@ -74,17 +74,48 @@ export const roundToTokenDecimals = (inputValue: string, tokenDecimals: number)
   // Combine the whole and adjusted decimal parts
   return whole + '.' + adjustedDecimals;
 };
+
 export enum Side {
   SUPPLY = 'supply',
   BORROW = 'borrow',
 }
-export const showSuperFestTooltip = (symbol: string, currentMarket: string, side?: Side) => {
+
+export type ExternalIncentivesTooltipsConfig = {
+  superFestRewards: boolean;
+  spkAirdrop: boolean;
+};
+
+export const showExternalIncentivesTooltip = (
+  symbol: string,
+  currentMarket: string,
+  side?: Side
+) => {
   const superFestRewardsEnabled = false;
+  const spkRewardsEnabled = true;
```

Comment:
is there a way to programmatically fetch this?

Reply:
On the forum it's written: "3.33m SPK tokens per month" but not end date is written. I'll find out more

Reply:
Because it's a pre-farming airdrop, it seems there is no end date for now, so I proposed to disable it manually when the time came (better than just put a random number of month for instance)